ehitchhiking-ui/src/app/shared/components/chat/chat.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="modal-wrapper">
+	<header class="modal-header">
+		<span class="modal-title">Chat</span>
+	</header>
+
+	<div class="chat" *ngIf="showDialog">
+		<div class="message-zone">
+			<div *ngFor="let msg of msgList" class="message-wrapper" [class.my-message]="msg.isMy">
+				<div class="person-info">
+					<img [src]="msg.avaSrc" alt="photo" class="image" />
+					<div class="date">{{ msg.time }}</div>
+				</div>
+				<div class="message">
+					<div class="text">{{ msg.text }}</div>
+				</div>
+			</div>
+		</div>
+
+		<form class="input-zone">
+			<mat-form-field class="input">
+				<input matInput #message maxlength="256" placeholder="Message" id="input-text" />
+				<mat-hint align="end">{{ message.value.length }} / 256</mat-hint>
+			</mat-form-field>
+			<button class="send-btn" mat-raised-button (click)="onInput(message.value); message.value = ''">
```

Comment:
pls move click hendle to separate function

Reply:
fixed

Reply:
done